### ISSUE TITLE
no-unused: exempt global augmentation

### ISSUE THREAD

**Issue by ajafff** [ajafff/tslint-consistent-codestyle#30]
```ts
declare global { // should not produce an error
}
```
